hat/app/org/hatdex/hat/phata/controllers/Phata.scala

```diff
@@ -89,14 +94,14 @@ class Phata @Inject() (
     }
   }
 
-  def hatLogin(name: String, redirectUrl: String) = indefiniteSuccessCaching {
-    Action { implicit request =>
-      val scheme = if (request.secure) { "https://" } else { "http://" }
-      val newRedirectUrl = s"$scheme${request.domain}/#/hatlogin?name=$name&redirect=${redirectUrl}"
-      logger.debug(s"Redirect url from ${request.uri}: $newRedirectUrl")
-      Redirect(newRedirectUrl)
-    }
-  }
+  //  def hatLogin(name: String, redirectUrl: String) = indefiniteSuccessCaching {
```

Comment:
Is there a reason for commented out code rather than just deleting it ?

Reply:
I wanted to get it tested in the staging environment before fully removing.